tests/pds/dataservice_test.go

```diff
@@ -82,54 +66,30 @@ func TestDataService(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	Step("get prerequisite params to run the pds tests", func() {
-		ControlPlaneURL := pdslib.GetAndExpectStringEnvVar(envControlPlaneURL)
-		Expect(ControlPlaneURL).NotTo(BeEmpty(), "ENV "+envControlPlaneURL+" is not set")
-
-		ClusterType := pdslib.GetAndExpectStringEnvVar(envClusterType)
-		Expect(ClusterType).NotTo(BeEmpty(), "ENV "+envClusterType+" is not set")
-
-		TargetClusterName := pdslib.GetAndExpectStringEnvVar(envTargetClusterName)
-		Expect(TargetClusterName).NotTo(BeEmpty(), "ENV "+envTargetClusterName+" is not set")
-
 		DeployAllDataService, err = pdslib.GetAndExpectBoolEnvVar(envDeployAllDataService)
 		Expect(err).NotTo(HaveOccurred())
 
-		DeployAllVersions, err = pdslib.GetAndExpectBoolEnvVar(envDeployAllVersions)
+		pdsparams := Inst().PDSParams
+		params, err = pdslib.ReadParams(pdsparams)
 		Expect(err).NotTo(HaveOccurred())
```

Comment:
change this to dash verifications

Reply:
In next automation sprint we will adapt to dash. In this manual sprint we are targeting only the parameter part to run tests from Jenkins